astropy/coordinates/baseframe.py

```diff
@@ -1260,17 +1260,15 @@ def is_transformable_to(self, new_frame):
         transformation between two objects of the same frame class but with
         different attributes.
         """
-
         new_frame_cls = new_frame if inspect.isclass(new_frame) else new_frame.__class__
         trans = frame_transform_graph.get_transform(self.__class__, new_frame_cls)
 
         if trans is None:
-            if new_frame_cls is self.__class__:
-                return 'same'
-            else:
-                return False
+            transformable = 'same' if new_frame_cls is self.__class__ else False
```

Comment:
I personally don't like ternary / inline statements, so I would keep this as
```suggestion
            if new_frame_cls is self.__class__:
                transformable = 'same'
            else:
                transformable = False
```
(but I don't have very strong feelings)

Reply:
If it's one-liner I prefer a ternary, but same here on the strong feelings.